src/pages/includes/gravatar/gravatar.ts

```diff
@@ -18,10 +17,4 @@ export class GravatarPage {
   width: number;
 
   constructor() {}
-
-  ngOnInit() {
-    if (typeof this.email === 'string') {
-      this.emailHash = Md5.hashStr(this.email.toLowerCase() || '');
```

Comment:
probably also want to delete `public emailHash` on line 8.

Reply:
+1 Thanks!